src/Discovery/DiscoveryListView.tsx

```diff
@@ -71,6 +72,18 @@ const DiscoveryListView: React.FunctionComponent<Props> = (props: Props) => {
           if (props.config.features.authorization.enabled) {
             disabled = (record[props.accessibleFieldName] !== AccessLevel.ACCESSIBLE) && (record[props.accessibleFieldName] !== AccessLevel.MIXED);
           }
+          // If enableFillRequestForm is true, allow users to check the checkbox without login
+          if (props.config.features.exportToWorkspace?.enableFillRequestForm) {
+            disabled = false;
+          }
+          // Check if the fillRequestFormCheckField is empty, and disable the checkbox if true
+          const fillRequestFormCheckField = props.config.features.exportToWorkspace?.fillRequestFormCheckField;
+          if (fillRequestFormCheckField) {
+            const fieldValue = record[fillRequestFormCheckField];
+            if (!fieldValue || fieldValue.length === 0) {
+              disabled = true;
+            }
+          }
```

Comment:
It would be better if this logic is put into the `if` block above

Reply:
should I merge it with this

https://github.com/urvi-occ/data-portal/blob/df82da937789dd1215e3e403dc7ded8e1fd9a972/src/Discovery/DiscoveryListView.tsx#L76-L78


or this

https://github.com/urvi-occ/data-portal/blob/df82da937789dd1215e3e403dc7ded8e1fd9a972/src/Discovery/DiscoveryListView.tsx#L88-L96

Reply:
into L76-78 should be more appropriate IMO

Reply:
Done